src/main/java/com/javadeobfuscator/deobfuscator/transformers/general/peephole/ConstantFolder.java

```diff
@@ -231,6 +235,14 @@ public boolean transform() throws Throwable {
                                             results.add(((Number) ((LdcFrame) frame.getComparators().get(0)).getConstant()).intValue() != ((Number) ((LdcFrame) frame.getComparators().get(1)).getConstant()).intValue());
```

Comment:
can't one define both int values used in all the ifs before all the if-else stuff? Would result in much cleaner code.